garage/tf/policies/continuous_mlp_policy_with_model.py

```diff
@@ -0,0 +1,117 @@
+"""
+This modules creates a continuous MLP policy network.
+
+A continuous MLP network can be used as policy method in different RL
+algorithms. It accepts an observation of the environment and predicts an
+action.
+"""
+from akro.tf import Box
+import tensorflow as tf
+
+from garage.misc.overrides import overrides
+from garage.tf.models.continuous_mlp_model import ContinuousMLPModel
+from garage.tf.policies.base2 import Policy2
+
+
+class ContinuousMLPPolicyWithModel(Policy2):
```

Comment:
Should this be DeterministicMLPPolicyWithModel? I think in the future we should rename it.

Reply:
We do have a DeterministicMLPPolicyWithModel.

I agree that the naming needs to be consistent and more precise. Except having `input_include_goal` as constructor argument, `ContinuousMLPModel` is same as `DiscreteMLPModel` with the output being scaled. 

Should we have a single `MLPModel` with output scale default to be 1? So that we can have the only one `MLPModel` to be used everywhere.